FRAuth/FRAuth/Error/WebAuthnError.swift

```diff
@@ -139,27 +179,27 @@ extension WebAuthnError: CustomNSError {
     }
 }
 
-extension WAKError {
+extension FRWAKError {
```

Comment:
@george-bafaloukas-forgerock does changing this name  to FRWAKError will break the backward compatibility? Like is any client will actually use or using this WAKError?

Reply:
Nope it wont as it's internal. I prefer to have the FR prefix tbh instead of a generic name